roles/advanced-core/advanced_dhcp_server/templates/dhcpd.subnet.conf.j2

```diff
@@ -71,11 +72,12 @@
   }
         {% endif %}
       {% endif %}
-    {% endfor %}
+    {% endfor -%}
```

Comment:
The minus sign should remove whitespace after the block, but there is no whitespace here?